Harvest.Web/Services/EmailService.cs

```diff
@@ -64,10 +67,14 @@ public async Task<bool> ProfessorQuoteReady(Project project)
 
         }
 
+        private async Task<string[]> FieldWorkersEmails()
+        {
+            return await _dbContext.Permissions.Where(a => a.Role.Name == Role.Codes.FieldManager).Select(a => a.User.Email).ToArrayAsync();
+        }

```

Comment:
Shouldn't this be filtering on `Roles.Codes.Worker`?

Reply:
Not worker, But I can change the name of the method in my next pr